### PR TITLE
Graph 357 add Message Handler

### PR DESCRIPTION
**Pull request by lavabyrd**
https://figmentio.atlassian.net/browse/GRAPH-357

### CODE REVIEW

**Review by kruspy — APPROVED** [review]
From what is said here https://docs.cosmos.network/master/basics/tx-lifecycle.html#state-changes the `DeliverTx` is correct for the message origin.

File: src/protocols/cosmos/manifest.graphql [hunk 1]
```diff
@@ -63,6 +63,16 @@ type EventHandler {
   handler: String!
 }
 
+type MessageOrigin {
+  DeliverTx
+}
+
+type MessageHandler {
+  message: String!
+  origin: MessageOrigin
```
Comment (pienkowb) [inline src/protocols/cosmos/manifest.graphql:72]: We don't need origin here, messages always come from transactions.

**Review by pienkowb — COMMENTED** [review]
Let's use a separate branch as base (the `cosmos` branch is used in the upstream PRs)

File: src/protocols/cosmos/manifest.graphql (hunk 1 above)
Comment (lavabyrd) [reply src/protocols/cosmos/manifest.graphql]: removed and updated the branch to cosmos-v1.1 for merging @pienkowb

File: src/protocols/cosmos/manifest.graphql [hunk 2]
```diff
@@ -63,6 +63,15 @@ type EventHandler {
   handler: String!
 }
 
+type MessageOrigin {
+  DeliverTx
+}
```
Comment (pienkowb) [inline src/protocols/cosmos/manifest.graphql:68]: We can remove the type too.

**Review by pienkowb — APPROVED** [review]
Looks good! 👍

**Pull request merged**